etc/orthography.tsv

```diff
@@ -0,0 +1,116 @@
+é	é/e	E	Vowel	6	U+00E9	"ʂɤ̀=pʰé,ʔé,θɯ̀=pʰé"	Brek_Kayaw,Geba_Karen,Kayah_Monu,Yintale
+ê	ê/e	E	Vowel	2	U+00EA	"êŋ,θǝ̂ŋ=pêŋ"	Pekon_Kayan
+ì	ì/i	I	Vowel	14	U+00EC	"swì,ǯɯ̄=θì=mì,ʂì"	Brek_Kayaw,Eastern_Kayah_Li,Geba_Karen,Kayah_Monu,Lahta_Kayan,Pekon_Kayan
+í	í/i	I	Vowel	14	U+00ED	"šwí,ǝ̄=θí-pàʔ,θí"	Brek_Kayaw,Bwe_Karen,Eastern_Kayah_Li,Geba_Karen,Kayah_Monu,Lahta_Kayan
+î	î/i	I	Vowel	2	U+00EE	"tʰwî,θwî"	Pekon_Kayan
+ò	ò/o	U	Vowel	11	U+00F2	"dɛ̀=tǝ̄ɓòʔ,ǝ̄=dòʔ,ǝ̄=sòʔ"	Brek_Kayaw,Eastern_Kayah_Li,Geba_Karen,Kayah_Monu
+ó	ó/o	U	Vowel	9	U+00F3	"cǝ̏=ʔʋó,ló=sá-sɛ̀ʔ,méʰ=pʰó"	Geba_Karen,Kayah_Monu,Lahta_Kayan,Pekon_Kayan,Yinbaw
+ù	ù/u	Y	Vowel	14	U+00F9	"kʰɔ́=mù=bé,lù-plī,tʰyù"	Brek_Kayaw,Eastern_Kayah_Li,Geba_Karen,Kayah_Monu,Lahta_Kayan,Yintale
+ú	ú/u	Y	Vowel	13	U+00FA	"kú=mʌ̀,pʰú,šú"	Brek_Kayaw,Bwe_Karen,Eastern_Kayah_Li,Geba_Karen,Kayah_Monu,Yintale
+û	û/u	Y	Vowel	1	U+00FB	"sǝ̀u=mû"	Pekon_Kayan
+ā	ā/a	A	Vowel	5	U+0101	"lā=ʔó,lāiʰ,pʰā"	Eastern_Kayah_Li,Pekon_Kayan,Yinbaw,Yintale
+ē	ē/e	E	Vowel	11	U+0113	"tʰwēi-klɔ̀,ʂwēi,ʆiē"	Bwe_Karen,Lahta_Kayan,Western_Kayah_Li,Yintale
+ě	ě/e	E	Vowel	1	U+011B	"sěin=bēi"	Yintale
+ī	ī/i	I	Vowel	4	U+012B	"=kʰwī,=ɕū=θǝ́=mī,θwī"	Bwe_Karen,Yintale
+ŋ	ŋ	N	Consonant	9	U+014B	"kān=ʔúŋ,lóu̯ŋ,ʔà=ʂǔŋ"	Lahta_Kayan,Pekon_Kayan,Yintale
+ō	ō/o	U	Vowel	6	U+014D	"fō,tʰò-pʰōʔ,ɕō"	Bwe_Karen,Geba_Karen,Lahta_Kayan
+š	ʃ	S	Consonant	5	U+0161	"šwéʔ-m̥èʔ,šwí,šú"	Brek_Kayaw,Geba_Karen,Lahta_Kayan
+ū	ū/u	Y	Vowel	9	U+016B	"=ɕū=θǝ́=mī,ʆū-qài̯,ʔà=tà=klū"	Brek_Kayaw,Bwe_Karen,Lahta_Kayan,Yintale
+ű	ű/u	Y	Vowel	1	U+0171	"pʰű"	Yinbaw
+ǔ	ǔ/u	Y	Vowel	1	U+01D4	"ʔà=ʂǔŋ"	Yintale
+ǝ	ə	E	Vowel	2	U+01DD	"cǝ=mīʰ,kǝ=nɔ̄=bȁ"	Western_Kayah_Li,Yinbaw
+ǯ	dʒ	T	Consonant	4	U+01EF	"ǯáu=mēin,ǯū-mì,ǯɯ̄=θì=mì"	Brek_Kayaw,Yintale
+ȁ	ȁ/a	A	Vowel	3	U+0201	"kǝ=nɔ̄=bȁ,pʰȁ,ȁ"	Western_Kayah_Li,Yinbaw
+ȉ	ȉ/i	I	Vowel	5	U+0209	"krwȉ,ʆwȉ,θwȉ"	Western_Kayah_Li,Yinbaw
+ȕ	ȕ/u	Y	Vowel	4	U+0215	"dȕ,tʰȕ"	Western_Kayah_Li,Yinbaw
+ɓ	ɓ	P	Consonant	4	U+0253	"lè=ɓà,ɕūɓá-pʰō,θò=ɓèʔ"	Bwe_Karen,Geba_Karen
+ɕ	ɕ	S	Consonant	4	U+0255	"=ɕū=θǝ́=mī,ɕá-,ɕō"	Bwe_Karen
+ɗ	ɗ	T	Consonant	1	U+0257	"ɗō"	Bwe_Karen
+ɯ	ɯ	I	Vowel	1	U+026F	"ɔ̋=lɯǝ̌ʰ"	Western_Kayah_Li
+ʂ	ʂ	S	Consonant	7	U+0282	"ʂì,ʂɛ̂,ʂɤ̀=pʰé"	Kayah_Monu,Yintale
+ʆ	ʑ	S	Consonant	5	U+0286	"ʆiē,ʆwì,ʆū-qài̯"	Bwe_Karen,Lahta_Kayan,Western_Kayah_Li
+ʋ	ʋ	W	Consonant	2	U+028B	"cǝ̏=ʔʋó,lʋȕʰ"	Yinbaw
+ʍ	ʍ	M	Consonant	1	U+028D	"rʋʰ=ʍe̋"	Yinbaw
+ʔ	ʔ	H	Consonant	34	U+0294	"ǝ̄=kʰwíʔ,ʔà=tà=klū,ʔēi"	Brek_Kayaw,Eastern_Kayah_Li,Geba_Karen,Kayah_Monu,Pekon_Kayan,Yinbaw,Yintale
+θ	θ	D	Consonant	20	U+03B8	"sù=θí=mìʔ,θwī,θǝ̂ŋ=pêŋ"	Brek_Kayaw,Bwe_Karen,Geba_Karen,Pekon_Kayan,Yinbaw
+cʰ	cʰ	C	Consonant	1	U+0063 U+02B0	"cʰó-mí"	Kayah_Monu
+e̋	e̋/e	E	Vowel	4	U+0065 U+030B	"rʋʰ=ʍe̋,šʰwe̋,θɯ̃̏...=pʰe̋"	Yinbaw
+iʰ	iʰ/i /h	I H	Vowel Consonant	3	U+0069 U+02B0	"cùʰ-kàiʰ,lāiʰ,pwáiʰ-trʋ̀"	Yinbaw
+i̋	i̋/i	I	Vowel	1	U+0069 U+030B	"tʰwi̋"	Western_Kayah_Li
+i̯	i̯/i	I	Vowel	4	U+0069 U+032F	"mì=pi̯à,ēi̯ŋ,ʆū-qài̯"	Eastern_Kayah_Li,Lahta_Kayan
```

Comment:
The i with little bridge under it is better rendered as a `j`. Same for u.

Reply:
:+1: